### DISCUSSION TITLE
Backup location for multiple clusters

### DISCUSSION THREAD

**Discussion by dkoci** [vmware-tanzu/velero#5876] [Community support Q&A]
Hi all.
I have started using velero. I am a bit confused about backup location in case of 3 k8s clusters. Is it better to have one S3 backup location for all clusters or 3 backup locations instead, one for each cluster?

Thank you

**Comment by shubham-pampattiwar** [comment]
So it depends on the use case. Are these 3 k8s clusters dependent on each other ?
- For instance, are you going to backup from one of these cluster and restore to one of these clusters, then in this case you will have to share the backup location.
- Are you using the k8s clusters independently ? And you want to take separate backups for each of these clusters ? Then it would make sense to have different buckets and different backup location altogether.

**Reply by dkoci** [reply]
We have 3 independent clusters (dev, test, prod). But sometimes we migrate some projects from dev to test too.
I think 3 buckets would be ok. It is possible to create multiple backup locations in case of migration from one cluster to another.
